yt/yt/server/node/exec_node/job_environment.cpp

```diff
@@ -987,40 +1010,42 @@ class TCriJobEnvironment
         PodDescriptors_.clear();
         PodSpecs_.clear();
         SlotCpusetCpus_.clear();
+        CpuLimit_ = cpuLimit;
 
-        {
-            std::vector<TFuture<TCriPodDescriptor>> podFutures;
-
-            podFutures.reserve(slotCount);
-            for (int slotIndex = 0; slotIndex < slotCount; ++slotIndex) {
-                auto podSpec = New<NCri::TCriPodSpec>();
-                podSpec->Name = Format("%v%v", SlotPodPrefix, slotIndex);
-                podSpec->Resources.CpuLimit = cpuLimit;
-                PodSpecs_.push_back(podSpec);
-                auto podFuture = Executor_->RunPodSandbox(podSpec);
-                podFutures.push_back(podFuture);
-
-                // Wait for the first slot, otherwise containerd runs parallel
-                // pull for sandbox_image (pause) if it is not cached yet.
-                if (slotIndex == 0) {
-                    WaitFor(podFuture)
-                        .ThrowOnError();
-                }
+        PodDescriptors_.resize(slotCount);
+        PodSpecs_.resize(slotCount);
+        SlotCpusetCpus_.resize(slotCount);
 
-                SlotCpusetCpus_.push_back(EmptyCpuSet);
-            }
+        // init fake slot to download resource once before running init slots concurrently
```

Comment:
Comments start with the capital letter and end with the dot as regular sentences